src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    int getArea();
```

Comment:
```suggestion
    double getArea();
```

I think area it's double value)